code/datums/brain_damage/split_personality.dm

```diff
@@ -238,5 +239,36 @@
 	if(objective)
 		to_chat(src, span_notice("Your master left you an objective: <b>[objective]</b>. Follow it at all costs when in control."))
 
+/datum/brain_trauma/severe/split_personality/blackout
+	name = "Blackout Drunk"
+	desc = "Patient brain is filled with alcohol and thus formed a new personality temporarily to survive."
+	scan_desc = "complete alcohol takeover"
+	gain_text = span_warning("You feel your consioucness fading...")
+	lose_text = "You have no memory of what happened while blacked out, try to trace your step and find out what happened while your were out."
+	poll_role = "black out character"
+	var/duration = 300 //Should last 5 minute
+
+/datum/brain_trauma/severe/split_personality/blackout/on_life(seconds_per_tick, times_fired)
+	if(current_controller == OWNER)
+		switch_personalities()
+	if(owner.stat == DEAD)
+		if(current_controller != OWNER)
+			switch_personalities(TRUE)
+		qdel(src)
+	if(duration == 0)
+		qdel(src)
```

Comment:
You totes should give the guy a killer headache mood event here.

Reply:
done!